erts/emulator/beam/external.c

```diff
@@ -1947,13 +1947,14 @@ static Eterm erts_term_to_binary_int(Process* p, Eterm Term, int level, Uint fla
 	    context_b = erts_create_magic_binary(sizeof(TTBContext),    \
                                                  ttb_context_destructor);   \
 	    context =  ERTS_MAGIC_BIN_DATA(context_b);			\
-	    sys_memcpy(context,&c_buff,sizeof(TTBContext));			\
+	    sys_memcpy(context,&c_buff,sizeof(TTBContext));		\
```

Comment:
The changes to this file don't look related to the rest of this commit, can you break them out?

Reply:
There's a magical number 3 used, which I copied to my code and it had to become 4 but it didn't and it created some nice segfaults. I gave it a reasonable name now so it won't bite anyone else.

Reply:
Both of these changes are good, I just figured they could have their own commit.